source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -31,6 +33,88 @@ There is currently support for the following device types within Home Assistant:
+| 8    | Duette, Top Down Bottom Up         | Top Down, Bottom Up             | Under Development |
+| 9    | Duette DuoLite, Top Down Bottom Up | Top Down, Bottom Up             | Under Development |
+| 18   | Silhouette                         | 90° Tilt when closed            | Under Development |
+| 23   | Silhouette                         | 90° Tilt when closed            | Confirmed         |
+| 38   | Silhouette Duolite                 | Dual Shade Blackout + 90° Tilt  | Under Development |
+| 42   | M25T Roller Blind                  | Bottom Up                       | Confirmed         |
+| 43   | Facette                            | 90° Tilt when closed            | Under Development |
+| 44   | Twist                              | 180° Tilt when closed           | Confirmed         |
+| 47   | Pleated, Top Down Bottom Up        | Top Down, Bottom Up             | Under Development |
+| 49   | AC Roller                          | Bottom Up                       | Confirmed         |
+| 51   | Venetian, Tilt Anywhere            | 180° Tilt Anywhere              | Under Development |
+| 54   | Vertical Slats, Left Stack         | 180° Tilt when closed, Vertical | Under Development |
+| 55   | Vertical Slats, Right Stack        | 180° Tilt when closed, Vertical | Under Development |
+| 56   | Vertical Slats, Split Stack        | 180° Tilt when closed, Vertical | Under Development |
+| 62   | Venetian, Tilt Anywhere            | 180° Tilt Anywhere, Vertical    | Under Development |
+| 65   | Vignette Duolite                   | Dual Shade Blackout             | Under Development |
+| 69   | Curtain, Left Stack                | 180° Tilt when closed, Vertical | Under Development |
+| 70   | Curtain, Right Stack               | 180° Tilt when closed, Vertical | Under Development |
+| 71   | Curtain, Split Stack               | 180° Tilt when closed, Vertical | Under Development |
+| 79   | Duolite Lift                       | Dual Shade Blackout             | Under Development |
+
+## Capabilities Information
+
+### Bottom Up
+
+These shades offer only the simple up/down movement of your conventional shades.
+
+### Top Down
+
+> :information_source: Full functionality not currently implemented
+These shades offer a unique movement which is inverse to your conventional shade where the shade is fixed to the floor and lowers from the roof.
+
+### Top Down, Bottom Up (TDBU)
+
+> :information_source: Full functionality not currently implemented
+TDBU shades consist of two rails controlled by two motors designated by Top and Bottom with fabric in between.
+The Top and Bottom can move independently of each other to cover different parts of the window but cannot pass the other.
+Two different entities that will be created for each blind: Top and Bottom.
+
+#### Top entity
+
+- 'Up/Open' will move the Top rail to the bottom of the window and set the Bottom rail to it's closed position.
```

Comment:
```suggestion
- 'Up/Open' will move the Top rail to the bottom of the window and set the bottom rail to its closed position.
```

Reply:
Or make them all `Top` and `Bottom`, but its confusing to have both

Reply:
I think they were all right to begin with - couldnt find any to fix.

When referring to the shades i used capatilised letters and when referring to the window it was lower case as i was indicating an item vs a position

ie - out of the 3 below the first felt the most natural - please let me know if you want all Top rail references changed - Perhaps it would read better with Rail capatilised too ?

Move the Top rail to the top of the window
Move the Top rail to the Top of the window
Move the top rail to the top of the window

Reply:
It's not a sticking point for me, only a suggestion. I'm happy with any of them.

Reply:
ok - if you want to take another run over it all your corrections are done